### ISSUE TITLE
plasflow and tensorflow=0.10.0rc0 packages not found on current channels via miniconda3 installation

### ISSUE THREAD

**Issue by Sfinks123** [smaegol/PlasFlow#9]
Hey Smaegol,

Thanks for updating the installation for PlasFlow. I am excited to use this novel tool to search for plasmids but ran into the following trying to install the new updates:

<img width="610" alt="screen shot 2018-06-01 at 4 17 10 pm" src="https://user-images.githubusercontent.com/39506567/40867219-7df8edbc-65b7-11e8-8d23-e34c6f592fc6.png">

Any thoughts on how to resolve the issue would be much appreciated. 

Thanks,
Sfinks

**Comment by smaegol** [comment]
On what platform are you installing PlasFlow (Mac/Linux)? Accidentally I forgot to commit the last update for README, now it is complete and provide another source for TensorFlow installation. Try with this and then invoke PlasFlow installation again and let me know if it helps.

**Comment by Sfinks123** [comment]
Hi Smaegol,

Thank you for all of your help! I was able to install PlasFlow successfully following the Conda-based installation instructions and with the updated README. Also, I ran PlasFlow on the test files provided and the test.plasflow_predictions.tsv output appears to match the predictions in Citrobacter_freundii_strain_CAV1321_scaffolds.fasta.PlasFlow.tsv.

Thank you again for all of your help,

Sfinks
